pkg/reconciler/claimbinding/api.go

```diff
@@ -125,18 +124,18 @@ func (a *APIManagedConnectionPropagator) PropagateConnection(ctx context.Context
 		}
 		cmcs.Data = mgcs.Data
 		meta.AddAnnotations(cmcs, map[string]string{
-			AnnotationKeyPropagateFromNamespace: mgcs.GetNamespace(),
-			AnnotationKeyPropagateFromName:      mgcs.GetName(),
-			AnnotationKeyPropagateFromUID:       string(mgcs.GetUID()),
+			resource.AnnotationKeyPropagateFromNamespace: mgcs.GetNamespace(),
+			resource.AnnotationKeyPropagateFromName:      mgcs.GetName(),
+			resource.AnnotationKeyPropagateFromUID:       string(mgcs.GetUID()),
 		})
 		return nil
 	}); err != nil {
 		return errors.Wrap(err, errCreateOrUpdateSecret)
 	}
 
-	meta.AddAnnotations(mgcs, map[string]string{
-		strings.Join([]string{AnnotationKeyPropagateToPrefix, string(cmcs.GetUID())}, SlashDelimeter): strings.Join([]string{cmcs.GetNamespace(), cmcs.GetName()}, SlashDelimeter),
-	})
+	k := strings.Join([]string{resource.AnnotationKeyPropagateToPrefix, string(cmcs.GetUID())}, resource.AnnotationDelimiter)
+	v := strings.Join([]string{cmcs.GetNamespace(), cmcs.GetName()}, resource.AnnotationDelimiter)
+	meta.AddAnnotations(mgcs, map[string]string{k: v})
```

Comment:
Thanks for cleaning this up :)

Reply:
Thanks! To be honest I think it was fine as is, but it started to feel pretty long after adding the `resource.` package name in a few places.